apps/indexer/ponder.schema.ts

```diff
@@ -124,16 +128,15 @@ export const transfer = onchainTable(
   }),
   (table) => ({
     pk: primaryKey({
-      columns: [table.transactionHash],
+      columns: [table.transactionHash, table.fromAccountId, table.toAccountId],
     }),
-    transfersTokenIdx: index().on(table.tokenId),
   }),
```

Comment:
The relation references are invalid. You're trying to match transactionHash + deltaMod against transactionHash + amount, and transactionHash + deltaMod against transactionHash + delegatedValue. These field combinations don't make logical sense and will cause relation lookup failures.